model/authorization/RemoteProcotoredSessionContext.php

```diff
@@ -44,4 +48,20 @@ public function getDeliveryExecutionId(): string
     {
         return $this->deliveryExecutionId;
     }
+
+    /**
+     * @throws SecurityException
+     */
+    public function consume(): void
+    {
+        if ($this->consumed) {
+            throw new SecurityException('Proctorio authorisation attempt has been already consumed');
+        }
+        $this->consumed = true;
```

Comment:
```suggestion
        $this->consumed = true;
```

This exception will never be thrown.